languages/go/exercises/concept/basic-strings/.docs/introduction.md

```diff
@@ -0,0 +1,14 @@
+# Introduction
+
+### Strings
+
+A `string` in Go is a sequence of `bytes`, which doesn't necessarily have to represent characters.
+That being said, `UTF-8` is a central part of `strings` in Go. It is easy to convert
+a string to `runes` (`UTF-8` characters) or iterate over `runes` in a string.
```

Comment:
We might want to talk about the fact that individual runes are not all the same length. Or is this better left for the blog link below?

Reply:
I'm not yet happy with this part as well. This exercise does not properly introduce runes. I wonder if it should or if it's too much. I'm currently thinking of doing that in a second part on more advanced strings. Also string iteration would be in that second part. So... not sure if we should even mention it here. I mostly wanted to note that UTF8 in Go is baked in -- so to say.

Reply:
I updated this. What do you think?